Userland/Libraries/LibWeb/CSS/Parser/Token.cpp

```diff
@@ -18,26 +18,26 @@ ErrorOr<String> Token::to_string() const
     case Type::EndOfFile:
         return String {};
     case Type::Ident:
-        return String::from_utf8(serialize_an_identifier(ident()));
```

Comment:
Not an issue with this PR but just a note these should have been `String::from_deprecated_string`

Reply:
Thanks! I didn't notice that was a thing before.